### ISSUE TITLE
How can I train the model without pre-trained weight specified?

### ISSUE THREAD

**Issue by salidw** [qfgaohao/pytorch-ssd#182]

**Comment by Jar-17** [comment]
I faced the same issue in training with mobilenetv3. I just get rid off the part that indicates the pre-trained model. The comment should like this 
"python train_ssd.py --datasets D:\pytorch-ssd\voc_v1 --validation_dataset D:\pytorch-ssd\voc_v1 --net mb3-large-ssd-lite  --scheduler multi-step --milestone 40,60  --lr 0.01 --t_max 100 --batch_size 36 --num_epochs 100 --base_net_lr 0.001"
